gst/mqtt/ntputil.h

```diff
@@ -25,3 +30,15 @@
  */
 int64_t
 ntputil_get_epoch (uint32_t hnums, char **hnames, uint16_t * ports);
+
+/**
+ * @brief Converting network byte order to host byte order.
+ * @note There is a problem in mocking ntohl in GMock, so the wrapper function is temporarily used.
+ * */
+uint32_t
+_convert_to_host_byte_order (uint32_t in);
```

Comment:
Good alternative solution!

At first, I wondered why you define the wrapper function instead of directly making a mock like `connect()`. So I made a simple test code and found out that a redefinition error occurs when making a mock. Your alternative solution looks reasonable since it is directly called `ntohl()`, which is a test target. :+1:

Reply:
Right. I also tried to solve the redefinition problem, but I couldn't find a way, so I used the wrapper function as an alternative.